pkg/collector/corechecks/cluster/kubernetes_openshift.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+// +build kubeapiserver
+
+package cluster
+
+import (
+	"errors"
+	"fmt"
+
+	osq "github.com/openshift/api/quota/v1"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// retrieveOShiftClusterQuotas lists and unmarshalls Openshift
+// ClusterResourceQuota objects from the APIserver
+func (k *KubeASCheck) retrieveOShiftClusterQuotas() ([]osq.ClusterResourceQuota, error) {
+	var url string
+	switch k.oshiftAPILevel {
+	case apiserver.OpenShiftAPIGroup:
+		url = "/apis/quota.openshift.io/v1/clusterresourcequotas/"
```

Comment:
Small nit, you can ignore but we could use a `var baseUrl  = "/apis/quota.openshift.io"`  `var oapiUrl = "/oapi"` and then `var clusterQuotaEndpoint = "/v1/clusterresourcequotas/"` ?